src/applications/appeals/shared/constants.js

```diff
@@ -124,6 +124,7 @@ export const REGEXP = {
   COMMA: /[, ]/g,
   DASH: /-/g,
   EMPTY_DATE: /(--|-00-00)/,
+  NON_DIGIT: /[^0-9]/g,
```

Comment:
Change if you'd like, but this could just be `/\D/g` 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape

Reply:
I'll update it in my next PR 😸